omniauth-keycloak.gemspec

```diff
@@ -9,25 +8,30 @@ Gem::Specification.new do |s|
   s.email       = 'maltmeyer@avarteq.de'
   s.homepage    = 'https://github.com/avarteqgmbh/omniauth-keycloak'
   s.license       = 'propitary'
-  s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
+  s.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
 
   s.files         = `git ls-files`.split("\n")
 
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
 
   s.require_paths = ['lib']
 
-  s.add_dependency "omniauth", ">= 2.0"
-  s.add_dependency 'omniauth-oauth2', '>= 1.8.0'
-  s.add_dependency 'jwt'
+  s.required_ruby_version = '>= 3.0.0'
+
   s.add_dependency 'httparty'
-  s.add_development_dependency 'activesupport'
-  s.add_development_dependency 'rails', '>= 6.1.7.1'
+  s.add_dependency 'jwt'
+  s.add_dependency 'nokogiri', '>= 1.16.5'
+  s.add_dependency 'omniauth', '>= 2.1.2'
+  s.add_dependency 'omniauth-oauth2', '>= 1.8.0'
+  s.add_dependency 'rack', '>= 2.2.8.1'
+
+  s.add_development_dependency 'activesupport', '>=  6.1.7.8'
+  s.add_development_dependency 'rails', '>=  6.1.7.8'
+  s.add_development_dependency 'rexml', '>= 3.2.7'
```

Comment:
According to dependabot the minimum version should be `3.3.2` https://github.com/avarteqgmbh/omniauth-keycloak/security/dependabot/61